### ISSUE TITLE
InvalidSignatureException: Credential should be scoped to correct service: 'application-autoscaling'.

### ISSUE THREAD

**Issue by ConradKurth** [aws/aws-sdk-go-v2#163]
Please fill out the sections below to help us address your issue.

2.0.0-preview.3

### Version of Go (`go version`)?

go version go1.10 darwin/amd64

### What issue did you see?

InvalidSignatureException: Credential should be scoped to correct service: 'application-autoscaling'. 

### Steps to reproduce

If you have have an runnable example, please include it.

```go
	cfg, err := external.LoadDefaultAWSConfig()
	if err != nil {
		panic(err)
	}
	cfg.Region = endpoints.UsWest2RegionID

	scale := applicationautoscaling.New(cfg)
	req := scale.RegisterScalableTargetRequest(&applicationautoscaling.RegisterScalableTargetInput{
		MaxCapacity:       aws.Int64(200),
		MinCapacity:       aws.Int64(90),
		ResourceId:        aws.String("service/foo/bar"),
		ScalableDimension: applicationautoscaling.ScalableDimensionEcsServiceDesiredCount,
		ServiceNamespace:  applicationautoscaling.ServiceNamespaceEcs,
	})
	_, err = req.Send()
	if err != nil {
		panic(err)
	}
```

**Comment by jasdel** [comment]
I think this issue is related to v1 SDK's aws/aws-sdk-go/#1009 and aws/aws-sdk-go#1850. And issue with this was fixed a while ago for `application-autoscaling` and I would of expected this to of been included in the V2 SDK. We'll take a look at this issue to see if implementing #153 can resolve this issue.

**Comment by jasdel** [comment]
@ConradKurth thanks for reaching out to us with this issue. I've merged in #181 which corrects how the SDK resolves services signing names. it will be tagged in the SDK's next release. Let us know if you have any additional questions or feedback about the SDK.
